CONTRIBUTING.md

```diff
@@ -72,6 +72,10 @@ For convenience here's a command for it:
 python -m venv .venv
 source .venv/bin/activate
 ```
+
```

Comment:
Why is this the case? I have it somewhere under my home dir ~/.virtual_envs and it's not related to the Danswer directory.

Reply:
Main point is that the virtual environment must not be inside the *danswer* project.  Will reword in #1631.